modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteOutOfMemoryPropagationTest.java

```diff
@@ -57,6 +71,9 @@
     /** */
     private CacheWriteSynchronizationMode writeSyncMode;
 
+    /** */
```

Comment:
Add javadoc.

Reply:
Seems this place is obvious enough to contain default javadoc.